src/testsuite/arith128_test_f64.c

```diff
@@ -1498,6 +1669,117 @@ test_double_cpsgn (void)
   return (rc);
 }
 
+int
+test_double_isfinite (void)
+{
+  vf64_t i;
+  vb64_t e, k;
+  int rc = 0;
+
+  printf ("\n%s double isfinite\n", __FUNCTION__);
+
+  i = (vf64_t){ 0.0, -0.0 };
+  e = (vb64_t) CONST_VINT128_DW(__DOUBLE_TRUE, __DOUBLE_TRUE);
```

Comment:
space between 'function' and open paren
_global comment_

Reply:
updated per comments